packages/components/src/DescriptionList/DescriptionList.test.tsx

```diff
@@ -14,6 +14,18 @@ it("renders an object as a list of key value pairs", () => {
   expect(container).toMatchSnapshot();
 });
 
+it("allows duplicate terms", () => {
+  const { container } = render(
+    <DescriptionList
+      data={[
+        ["Issued", "2018-12-08"],
+        ["Issued", "2019-01-06"],
+      ]}
+    />,
+  );
+  expect(container).toMatchSnapshot();
+});
```

Comment:
quick question, is this meant to test the key problem? Or just that you can put duplicate terms?

Reply:
Both, in a way. This is just to check that there are no warnings when duplicate terms are used, which was what happened before my change. However, this test wouldn't have explicitly failed before, either. It just would have caused a warning message. I added a check to ensure that the console isn't issuing a warning. Does that make sense? Is this a good approach? Sorry, still ramping up my React skills.